docs/more-resources.md

```diff
@@ -40,4 +40,7 @@ React Native has a community of thousands of developers like you making content,
 
 Can’t find what you’re looking for in the Core Components? Visit [React Native Directory](https://reactnative.directory) to find what the community has been creating.
 
-Interested in making your own Native Component or Module? Making modules for your own use case and sharing them with others on NPM and GitHub helps grow the React Native ecosystem and community! Read the guides to making your own Native Modules ([Android](native-modules-android.md), [iOS](native-modules-ios.md)) and Native Components ([Android](native-components-android.md), [iOS](native-components-ios.md)).
+:::caution
```

Comment:
shouldn't we refer to the new guides to TM and Fabric components once we have them, rather than pointing the users to the old legacy docs?

Reply:
I've left this `::caution` more as a way to signal to us that this needs additional work.  My preferences is to move these docs under `legacy/` now then to update as we introduce the new docs.